internal/controllers/analyzer/analyzer.go

```diff
@@ -171,6 +171,12 @@ func (a *Analyzer) runAnalysis() (totalVulns int, err error) {
 	if err != nil {
 		return 0, err
 	}
+
+	if a.config.EnableGitHistoryAnalysis {
+		logger.LogWarnWithLevel(messages.MsgWarnGitHistoryEnable)
+		print("\n")
```

Comment:
This print is really necessary? I would prefer always use the logger to print something. If this is really necessary change to fmt.Println.

Reply:
Space needed so that the message is not too close to the loading.